2024/incoming/perl-version-bumper.pod

```diff
@@ -0,0 +1,619 @@
+corresponding feature bundle. For example:
+
+=begin perl
+
+   use v5.10;
+
+=end perl
+
+is equivalent to:
+
+=begin perl
+
+   use feature ':5.10';
+
+=end perl
+
+and will enable the C<say>, C<state> and C<switch> features. The
+compiler will now understand the corresponding keywords, and behave
+accordingly. Error messages might differ a lot when a feature is
+enabled or not.
+
+Note that adding a subversion (e.g. C<use v5.36.3>) will have no effect
+on the bundles loaded (feature bundles are guaranteed to be the same for
+all sub-versions). It will of course have an effect on the interpreter
+version check described above.
+
+In addition to the implicit loading of features, C<use v5.12;> or greater
+will enable C<strict>, C<use v5.36;> will enable C<warnings> and C<use
+v5.40;> will import L<builtin> functions (using a similar version bundle
+scheme: see L<builtin/Version-Bundles>).
+
+Note that this is a lexical pragma, meaning that you could declare
+different version bundles for different parts of your code. For
+consistency, it's really recommended that you pick one version, and
+stick with it for the entire file. In fact, a future release of C<perl>
+will disallow changing the version once one has been declared. So,
+really, don't do that.
+
+The features included in bundles fall in two broad categories: new
+features, and deprecated features. Loading a bundle will perform the
+following two operations in a single line of code:
+
+=head4 Enable modern Perl features
+
+Perl features are generally used to introduce experimental features
+without making them part of the "official" language. Features become
+official when they are included in a feature bundle. Failed experiments
+can be removed safely if they never come out of the experimental phase.
+
+For example, the C<signatures> feature was introduced in C<perl> v5.20,
+and remained experimental until v5.34. That meant that, as long as the
+C<perl> binary was more recent than v5.20, one could use signatures in
+their code with C<use feature 'signatures';>. That specific feature has
+been added to all bundles since C<:5.36>, which means that a single
+declaration (C<use v5.36>, or any later version) allows one to write
+Perl subroutines using signatures.
+
+Not all new features have to wait that long to become part of the
+language: C<module_true> was introduced in C<perl> v5.38, and was
+immediately added to the C<:5.38> feature bundle.
+
+=head4 Deprecate discouraged features
+
+Bundles have also been used to deprecate and disable features that have
+become discouraged. These are made part of the C<:default> feature bundle,
+and not included in later bundles.
+
+Doing it this way makes it possible to preserve the behaviour of
+ancient, unmaintained, scripts and modules. Even if they load some
+feature bundle (via C<use VERSION>), any deprecated feature they might
+use will also be (retroactively) included in that bundle, preserving
+backward compatibitlity.
```

Comment:
This is inaccurate.  *Discouraged* features might be present.  *Deprecated* features are doomed.  It doesn't matter what version bundle you use, assigning to `$[` just doesn't work anymore.